src/index.js

```diff
@@ -0,0 +1,171 @@
+import './style.css';
+
+const list = JSON.parse(localStorage.getItem('list')) || [{
+  descreption: 'first',
+  completed: false,
+},
+{
+  descreption: 'second',
+  completed: false,
+},
+{
+  descreption: 'third',
+  completed: false,
+}];
```

Comment:
- [ ] There should be no hardcoded to-do objects as per the project requirement in this milestone. Kindly remove them. :pray: